### ISSUE TITLE
Add show card info button to anki reviewer

### ISSUE THREAD

**Issue by evandrocoan** [ankitects/anki#1347]
There is some time I wanted to easily see the current card stats (without having to open the browser window). So, make this quick patch on my [anki fork](https://github.com/evandroforks/anki/commit/164ae25bbb23be1f8385c1c867cb8348d2a1018d). The button would show up here:

![image](https://user-images.githubusercontent.com/5332158/131263345-a91b3106-721b-4a28-adef-ee094ba8879e.png)

![image](https://user-images.githubusercontent.com/5332158/131263519-fe39b2b6-90b4-4625-80f0-0ec56425c983.png)

As of now, it is not proper for pull-request because I just glue together a bunch of imports and copied code. If someone is interested and this button is accepted, a pull-request could be opened properly adding the button (without duplicating the code from `browser.py` and properly importing things).

```patch
 qt/aqt/reviewer.py | 46 ++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 46 insertions(+)

diff --git a/qt/aqt/reviewer.py b/qt/aqt/reviewer.py
index 500c5dc4..f0749974 100644
--- a/qt/aqt/reviewer.py
+++ b/qt/aqt/reviewer.py
@@ -120,6 +120,50 @@ class Reviewer:
     # Audio
     ##########################################################################
 
+    def cardInfo(self) -> None:
+        from anki.stats import CardStats
+        from aqt.qt import QDialog
+        from PyQt5 import QtCore, QtGui, QtWidgets
+        from aqt.webview import AnkiWebView
+        from aqt import QDialogButtonBox, qconnect, Qt
+        from aqt.utils import saveGeom, restoreGeom
+
+        cs = CardStats(self.mw.col, self.mw.reviewer.card)
+        info = cs.report(include_revlog=True)
+        reps = ""
+
+        class CardInfoDialog(QDialog):
+            silentlyClose = True
+
+            def __init__(self, browser, *args, **kwargs):
+                super().__init__(browser, *args, **kwargs)
+                self.browser = browser
+
+            def reject(self):
+                saveGeom(self, "revlog")
+                return QDialog.reject(self)
+
+        card_info_dialog = CardInfoDialog(None)
+        icon = QtGui.QIcon()
+        icon.addPixmap(
+            QtGui.QPixmap(":/icons/anki.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off
+        )
+        card_info_dialog.setWindowIcon(icon)
+
+        l = QtWidgets.QVBoxLayout()
+        l.setContentsMargins(0, 0, 0, 0)
+        w = AnkiWebView(title="browser card info")
+        l.addWidget(w)
+        w.stdHtml(info + "<p>" + reps, context=card_info_dialog)
+        bb = QDialogButtonBox(QDialogButtonBox.Close)
+        l.addWidget(bb)
+        qconnect(bb.rejected, card_info_dialog.reject)
+        card_info_dialog.setLayout(l)
+        card_info_dialog.setWindowModality(Qt.WindowModal)
+        card_info_dialog.resize(500, 400)
+        restoreGeom(card_info_dialog, "revlog")
+        card_info_dialog.show()
+
     def replayAudio(self) -> None:
         if self.state == "question":
             replay_audio(self.card, True)
@@ -290,6 +334,7 @@ class Reviewer:
             # ("m", self.showContextMenu),
             ("r", self.replayAudio),
             (Qt.Key_F5, self.replayAudio),
+            ("Ctrl+i", self.cardInfo),
             ("Ctrl+1", lambda: self.setFlag(1)),
             ("Ctrl+2", lambda: self.setFlag(2)),
             ("Ctrl+3", lambda: self.setFlag(3)),
@@ -739,6 +784,7 @@ time = %(time)d;
             # [_("Delete Note"), "Ctrl+Delete", self.onDelete],
             # [_("Options"), "O", self.onOptions],
             None,
+            [_("Card Info"), "Ctrl+I", self.cardInfo],
             [_("Replay Audio"), "R", self.replayAudio],
             [_("Pause Audio"), "5", self.on_pause_audio],
             [_("Audio -5s"), "6", self.on_seek_backward],
-- 
2.28.0.windows.1
```

**Comment by hikaru-y** [comment]
It looks like this has already been implemented in commit 5bd2dc6.

**Comment by evandrocoan** [comment]
Cool. Thanks for pointing it out.
